engine/api/migrate/stage_conditions.go

```diff
@@ -0,0 +1,70 @@
+package migrate
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/engine/api/cache"
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/log"
+)
+
+type prerequisitesDB struct {
+	sdk.Prerequisite
+	PipelineStageID int64 `json:"pipeline_stage_id"`
+}
+
+func StageConditions(store cache.Store, DBFunc func() *gorp.DbMap) error {
```

Comment:
do you need really need `DBFunc func() *gorp.DbMap` ? It seems a `gorp.SqlExecutor`would be enough